evalml/tests/component_tests/test_components.py

```diff
@@ -926,7 +925,7 @@ def test_transformer_transform_output_type(X_y_binary):
                 assert transform_output[0].shape == X.shape
                 assert transform_output[1].shape[0] == X.shape[0]
                 assert len(transform_output[1].shape) == 1
-            elif "sampler" in component.name:
+            elif isinstance(component, BaseSampler):
```

Comment:
Using this as a way to check if we're testing for sampler component rather than using name :)